tests/test_bls.py

```diff
@@ -238,3 +240,38 @@ def test_multi_aggregation(msg_1, msg_2, privkeys_1, privkeys_2):
         signature=aggsig,
         domain=domain,
     )
+
+
+def test_multi_multi():
+    domain = 0
+    validator_indices = tuple(range(10))
+    privkeys = tuple(2**i for i in validator_indices)
+    pubkeys = [privtopub(k) for k in privkeys]
+
+    class Attestation:
+        def __init__(self, msg_1, msg_2):
+            msg_1_validators = sample(validator_indices, 3)
```

Comment:
Anytime we use randomness in tests we should be using the mechanisms provided by `hypothesis` since they allow us to reproduce failures when they occur.  Otherwise failures due to random data will be opaque and we won't be able to recreate the conditions that produced them.

See: https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.random_module

Reply:
Removed randomness for now. Planning to rewrite bls tests with pytest fixtures and hypothesis.